packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react";
+import { render, screen, axe } from "reakit-test-utils";
+import AlertTest from "..";
+
+test("a11y", async () => {
+  const { baseElement } = render(<AlertTest />);
+  expect(await axe(baseElement)).toHaveNoViolations();
```

Comment:
<pre>
FAIL packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx
  ● a11y

    expect(received).toHaveNoViolations(expected)

    Expected the HTML found at $('dialog') to have no violations:

    <dialog role="alert" open="">Alert</dialog>

    Received:

    "ARIA role must be appropriate for the element (aria-allowed-role)"

    Fix any of the following:
      ARIA role alert is not allowed for given element

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.0/aria-allowed-role?application=axeAPI

       5 | test("a11y", async () => {
       6 |   const { baseElement } = render(<AlertTest />);
    >  7 |   expect(await axe(baseElement)).toHaveNoViolations();
         |                                  ^
       8 | });
       9 | 
      10 | test("markup", () => {

      at Object.<anonymous> (packages/reakit/src/Alert/__examples__/Alert/__tests__/index-test.tsx:7:34)
</pre>